src/gmail.js

```diff
@@ -2532,9 +2547,15 @@ var Gmail = function(localJQuery) {
                         var removedNodes = mutation.removedNodes;
                         for (var j = 0; j < removedNodes.length; j++) {
                             var removedNode = removedNodes[j];
+                            if (removedNode.className === "agh" && removedNode.querySelector("div[data-hovercard-id]")) { // contains recipient in peoplekit
+                                let observer = api.tracker.dom_observer_map["afV"];
+                                let handler = api.tracker.dom_observers.recipient_change.handler;
+                                api.observe.trigger_dom(observer, $(mutation.target), handler);
+                            }
+
                             if (removedNode.className === "vR") {
```

Comment:
I'm not sure how that could happe, but just to prevent the same event from triggering twice, would it make sense to make this an `else if` instead?

It would also make it clearer in code that those 2 events/code-paths should never occur at the same time.

Reply:
Great!